app/views/layouts/_navbar.html.haml

```diff
@@ -29,7 +29,7 @@
                 %i.fa.fa-file-text
                 %span My Proposals
 
-          - if (current_user.reviewer? || current_user.organizer?) && event
+          - if (current_user.reviewer? || current_user.organizer?) && event && event_staff?
```

Comment:
Wouldn't this be duplicative? The event_staff checks event.event_teammates which have a required role?